src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  place-items: center;
+
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+
+  padding: 32px 16px 16px;
+  border-radius: $border-radius-main;
+  width: $card-width;
+
+  &__img {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-gray-light;
+
```

Comment:
please removed these blank spaces, they are redundant here and make css file bigger for no reason, please do it in all css files